playbooks/common/openshift-cluster/initialize_facts.yml

```diff
@@ -8,3 +8,5 @@
       role: common
       local_facts:
         hostname: "{{ openshift_hostname | default(None) }}"
+  - set_fact:
+      openshift_docker_hosted_registry_network: "{{ hostvars[groups.oo_first_master.0].openshift.common.portal_net }}"
```

Comment:
I don't think we can primarily because it's the first master's fact we need, I think that implies we'd have to pass in the fact from the first master for all other systems, then set the fact on in openshift_facts on those systems. At that point I'm thinking it makes the most sense to be defined in the playbook/orchestration layer.

I've been thinking about this more since and I'm still feeling like we need a centralized place that sorts out *our* inventory vars, and makes sure they're set sanely throughout the roles. Roles should still have sensible defaults but in this case, it's a problem with that sensible default accidentally getting used when we don't want it to, because passing it individually for each role use is so fragile. This, or something like it, would then become a centralized yml you can include in any of our playbooks to initialize, evaluate groups would be a good candidate to merge into here too likely.